src/configure/resources/tracePoints.ts

```diff
@@ -0,0 +1,34 @@
+export class TracePoints {
```

Comment:
I think the first set of names are not very much acting as tracepoints. these are actionNames or some eventName.  The second set of failures are good. You can rethink about it categorizing action events separate from tracepoints. Call it EventName or something.

Reply:
+1 You can call it CIPoints

Reply:
segregated them